hbase-server/src/main/java/org/apache/hadoop/hbase/master/webapp/RegionReplicaInfo.java

```diff
@@ -102,6 +117,27 @@ public ServerName getServerName() {
     return serverName;
   }
 
+  public String getTransitioningOnServerName() {
+    return transitioningOnServerName;
+  }
+
+  public String getMergeRegionName() {
+    return (mergeRegionInfo != null)
+      ? mergeRegionInfo.stream()
+                       .map(RegionInfo::getRegionName)
+                       .map(Bytes::toStringBinary)
+                       .collect(Collectors.joining(", "))
+      : "";
```

Comment:
nit: `? :` is good to use but here since we have some good computation, it might be preferable to return empty string if regionInfo is null initially. Better readability? WDYT?

Reply:
This has been simplified with nullable collection. So no `? :` is required here then.